items.json

```diff
@@ -40113,7 +40113,7 @@
     "weight": 3.1,
     "tier": 7.56281948,
     "rank": 3,
-    "requirement": 15,
+    "requirement": 18,
```

Comment:
Can you manually remove these changes to str requirement on the xbows? This should be fixed when #473 is merged.